src/components/Table/Table.scss

```diff
@@ -0,0 +1,25 @@
+@use '@gravity-ui/uikit/styles/mixins';
```

Comment:
To be able to use `@use` or `@import` with node_modules styles you have to modify `gulpfile.js`. Add `includePaths: ['node_modules']`:
```js
    return src(['src/components/**/*.scss', '!src/components/**/__stories__/**/*'])
        .pipe(
            sass({
                includePaths: ['node_modules'],
            }).on('error', sass.logError),
        )
```